### ISSUE TITLE
Can't connect (LAN)

### ISSUE THREAD

**Issue by darkphoenixfox** [davidzech/TC5Ever#2]
PC1 : Windows 11, Games runs fine by itself with or without sinden Gun, IP: 192.168.0.22
Game started with `@start TimeCrisisGame-Win64-Shipping.exe -NOINI -Language=JPN -netcoophost=1 -playside=1`
DLL not present

PC2: Windows 10, Game also runs fine by itself, IP:192.168.0.183
Game started with `@start TimeCrisisGame-Win64-Shipping.exe -NOINI -Language=JPN -netcoophost=2 -playside=2 -address=192.168.0.22`
DLL is present in the game folder.

When I run the game on PC2 I get the console window with the following text:
```
address: 192.168.0.22
playerside: 1
detours attached
connect() hook called
connect() hook called
connect() hook called
connect() hook called
[...]
```
The connect() hook called line keeps repeating.

PC1 game works as P1 (RED) and PC2 game works as P2 (Blue)

They don't see each other
![image](https://user-images.githubusercontent.com/21360801/153658696-d8958760-2b62-4f8a-bee3-7dd8754ab725.png)

I can ping each PC from the other end. Firewall is off and ports are open.

**Comment by davidzech** [comment]
I think I typod the argument, it should be "-playerside=2" and you should see it reflected in the console window.

**Comment by darkphoenixfox** [comment]
Testing that now! Thanks

**Comment by darkphoenixfox** [comment]
If I replace `-playside=2` with `-playerside=2`, the console still says `playerside=1`  and in-game the game is set to player 1 (RED).

The game is definitely looking for -playside to decide between Red and Blue.

**Comment by davidzech** [comment]
Weird, I'll take a look soon. 

In the meantime, can you post me your start scripts?

**Comment by darkphoenixfox** [comment]
Start P2.bat
```
@start TimeCrisisGame-Win64-Shipping.exe -NOINI -Language=JPN -netcoophost=2 -playerside=2 -address=192.168.0.22
```

Start P1.bat
```
@start TimeCrisisGame-Win64-Shipping.exe -NOINI -Language=JPN -netcoophost=1 -playerside=1
```

You are more than welcome to join the Sinden discord if you want to discuss this live:
https://discord.gg/bUy9gCzA

**Comment by davidzech** [comment]
Yeah this is a me problem, I'll fix shortly

**Comment by darkphoenixfox** [comment]
Thanks for checking this.

**Comment by davidzech** [comment]
Fixed by 880e1358104ab09ac96216a670a5e993266be04e
